tests/integ/test_dataframe.py

```diff
@@ -1478,6 +1479,41 @@ def test_create_dataframe_with_semi_structured_data_types(session):
     )
 
 
+@pytest.mark.skipif(not is_pandas_available, reason="pandas is required")
+def test_create_dataframe_with_pandas_df(session):
```

Comment:
Do we need to port some changes to sp connector too to support `use_logical_type` in stored proc?

Reply:
That's right. We need to port this to sproc too

Reply:
https://github.com/snowflakedb/Stored-Proc-Python-Connector/pull/140